membership.md

```diff
@@ -0,0 +1,91 @@
+Membership and Commit Rights
+============================
+
+This document covers processes to be granted membership in the Dask
+GitHub organization and commit rights to Dask repositories.
+
+
+Owners
+------
+
+There is a set of individuals who have ownership privileges over the entire
+GitHub organization.  As a result, they have commit rights to all repositories
+and are involved in creating and removing members.  These individuals are
+listed at https://github.com/orgs/dask/people .
+
+We ask that owners ensure that their membership is public.
+
+New owners are made by lazy consensus of previous owners.  All owners will be
+made aware of a new candidate.  Any -1 vote bars a candidate.  We ask that a
+majority of owners respond for quorum, and that those votes come from people
+employed/associated to at least three institutions.
+
+
+Members
+-------
+
+New members can be made by any owner if the following criteria are met:
+
+1.  At least two owners from separate employers agree membership is a good idea
+2.  The new member has supported the project several times, either through code
+    or otherwise, and has demonstrated a willingness to help solve other
+    people's problems, not just their own.
+3.  The new member demonstrates some degree of familiarity of open source
+    practices and online courtesy.
+
+Write privileges
+----------------
+
+Individuals can be given write privileges to any repository by an owner by the
+above conditions and the following:
+
+1.  The lead maintainer of that repository thinks it's a good idea
```

Comment:
How do we decide who is the lead maintainer?

Reply:
I think that in practice we generally just know.  Happy to add something here if you have a suggestion.

Reply:
I think it was "lead" that was confusing to me. For instance I don't know the lead maintainer of dask-jobqueue, but know a few people that are heavily involved. Have pushed some text to relax this to maintainers instead of lead. Though am happy to discuss further as needed.